book/content/rdm/rdm.md

```diff
@@ -101,6 +102,33 @@ To avoid loosing your data, you should follow good backup practices.
 
 The more important the data and the more often the datasets change, the more frequently you should back them up.
 If your files take up a large amount of space and backing up all of them would be difficult or expensive, you may want to create a set of criteria for when you back up the data. This can be part of your data management plan.
+
+<a name=Spreadsheets"></a>
+### Data organisation in spreadsheets
+
+Spreadsheets, such as Microsoft Excel files, are commonly used to collect, store, manipulate, analyse, and share research data. Spreadsheets are convenient and easy-to-use tools for these tasks but are not amenable to reproducibility if used as dynamic documents. There is a collection of [horror-stories](http://www.eusprig.org/horror-stories.htm) that document the many ways that the use of spreadsheets have scuppered analyses due to unexpected behaviour or error-prone editing processes. Some of these mishaps are not unique to spreadsheets, but [many are](https://doi.org/10.1186/s13059-016-1044-7). 
```

Comment:
Thanks @wjchulme! this is great!

Please can you put each new sentence on a new line? I know the rest of the file doesn't do this but it would be great to start fixing forwards on this. (It helps with version control so we can see which full sentences have changed 😸).

Is there a reference list at the end of this page? Is this paper included in that list?

Reply:
Put in new lines for the whole chapter where needed.